### PR TITLE
Adding folder wallet

### PR DESCRIPTION
**Pull request by vscatarino**

### CODE REVIEW

File: host/vite.config.ts [hunk 1]
```diff
@@ -1,7 +1,22 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react-swc'
+import federation from '@originjs/vite-plugin-federation'
```
Comment (sandhilt) [inline host/vite.config.ts:3]: Did you add federation in this project? Impressive.

File: wallet/src/App.js
Comment (fabiooshiro) [inline wallet/src/App.js:1]: Having both App.js and App.tsx in the same directory will create problems.

File: wallet/src/components/Greetings.js
Comment (fabiooshiro) [inline wallet/src/components/Greetings.js:1]: same here...

**Review by fabiooshiro — CHANGES_REQUESTED** [review]

File: host/vite.config.ts (hunk 1 above)
Comment (vscatarino) [reply host/vite.config.ts:3]: Yes, federation is included.

**Review by fabiooshiro — APPROVED** [review]

**Pull request merged**